Mage.Tests/src/test/java/org/mage/test/cards/single/khm/MaskwoodNexusTest.java

```diff
@@ -0,0 +1,64 @@
+package org.mage.test.cards.single.khm;
+
+import mage.abilities.keyword.HexproofAbility;
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+import mage.abilities.keyword.HexproofAbility;
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+public class MaskwoodNexusTest extends CardTestPlayerBase {
+
+
+    private static final String maskwood = "Maskwood Nexus";
+    private static final String magda = "Magda, Brazen Outlaw";
+    private static final String pMonk = "Pinnacle Monk";
+    private static final String jadeOrb = "Jade Orb of Dragonkind";
+
+    @Test
+    public void magdaSearch() {
+        addCard(Zone.BATTLEFIELD, playerA, magda);
+        addCard(Zone.BATTLEFIELD, playerA, maskwood);
+        addCard(Zone.LIBRARY, playerA, pMonk);
+        // create 5 treasures
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain", 10);
+        addCard(Zone.BATTLEFIELD, playerA, "Treasure Vault", 1);
+        activateAbility(1, PhaseStep.PRECOMBAT_MAIN, playerA, "{X}{X}, {T}, Sacrifice");
+        setChoice(playerA, "X=5");
+
+        // activate Magda and select Pinnicle Monk
+        activateAbility(1, PhaseStep.POSTCOMBAT_MAIN, playerA, "Sacrifice five Treasures:");
+
+
```

Comment:
set strict choose mode true

Reply:
When setting strict mode with Magda it fails for some reason but when I do the exact same test in game its successful.
Any ideas why?

Reply:
No idea based on that, though it could help if you showed what the failure is